### PR TITLE
IDEM-216

### PR DESCRIPTION
**Pull request by JasminFarhi**
Was not sure which fields required it, so just added it to... Mobile, TFN, and House number for now. :) 

Screenshot_20220704-115333
Screenshot_20220704-115341
Screenshot_20220704-115349
Screenshot_20220704-115443

### CODE REVIEW

**Review by xiangzhe0812 — DISMISSED** [review]

**Comment by bitcoinbrisbane** [comment]
Mobile needs to be the mobile and email the email keypad

File: app/src/data/claims.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ const claims: Claim[] = [
     title: "Mobile",
     description: "Users mobile number",
     verificationAction: "action",
-    fields: [{ id: "mobileNumber", title: "Mobile number", type: "text" }],
+    fields: [{ id: "mobileNumber", title: "Mobile number", type: "number" }],
```
Comment (bitcoinbrisbane) [inline app/src/data/claims.ts:72]: Needs to be phone keyboard

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
See comments

**Review by bitcoinbrisbane — APPROVED** [review]
LGTM

**Review by xiangzhe0812 — APPROVED** [review]

**Pull request merged**